app/routes/circulars.correction.$circularId.tsx

```diff
@@ -37,7 +37,8 @@ export async function loader({
   try {
     existingRequest = await getChangeRequest(parseFloat(circularId), user.sub)
   } catch (error) {
-    console.info('No existing request found')
+    const err = error as Response
+    if (err.status !== 404) throw err
```

Comment:
Avoid type assertions. Use type narrowing wherever possible.

```suggestion
    if (!(err instanceof Response && err.status === 404)) throw err
```